src/protocols/secure_channel/CASESession.cpp

```diff
@@ -225,7 +226,7 @@ CASESession::ListenForSessionEstablishment(uint16_t localSessionId, FabricTable
 }
 
 CHIP_ERROR CASESession::EstablishSession(const Transport::PeerAddress peerAddress, FabricInfo * fabric, NodeId peerNodeId,
-                                         uint16_t localSessionId, ExchangeContext * exchangeCtxt,
+                                         SessionHolder & secureSessionHolder, ExchangeContext * exchangeCtxt,
```

Comment:
`SessionHolder &&`

Reply:
As suggested by @mrjerryjohns , PASESession and CASESession are now doing the allocation themselves, so this doesn't need to be passed in the first place.